charts/sysdig/values.yaml

```diff
@@ -32,6 +32,38 @@ resources:
     cpu: 3000m
     memory: 3072Mi
 
+# Specify a resource profile.
+#
+# Available options are:
+# * small
+#   requests:
+#     cpu: 1000m
+#     memory: 1024Mi
+#   limits:
+#     cpu: 1000m
+#     memory: 1024Mi
+#
+# * medium
+#   requests:
+#     cpu: 3000m
+#     memory: 3072Mi
+#   limits:
+#     cpu: 3000m
+#     memory: 3072Mi
+#
+# * large
+#   requests:
+#     cpu: 5000m
+#     memory: 6144Mi
+#   limits:
+#     cpu: 5000m
+#     memory: 6144Mi
+#
+# * custom
```

Comment:
A suggestion: Put this whole section under comments to make it easier for customers to use
~~
 If you want to define a custom resource profile, fill in the appropriate values here
 ```yaml
 requests:
  cpu: <cpu requests>m
  memory: >memory requests>Mi
 limits:
  cpu: <cpu limits>m
  memory: <memory limits>Mi
```
~~~